api/tests/LocalAuthBearerTokenTest.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+use App\Services\LocalAuthBearerTokenService;
+use App\Services\Contracts\BearerTokenServiceInterface;
+
+class LocalAuthBearerTokenTest extends TestCase
+{
+    /**
+     * @var BearerTokenServiceInterface
+     */
+    protected $service_provider;
+
+    protected function setUp(): void
+    {
+        // generate keys and tokens for testing at https://jwt.io/#debugger-io
+        // make sure you set algorithm to RS256
+        $issuer = 'http://test.com';
+        $publicKey = file_get_contents('tests/resources/key1-cert.pem');
+        if(!$publicKey)
+            throw new Exception('Failed to load public key from file');
+        $timeZone = 'UTC';
+
+        $this->service_provider = new LocalAuthBearerTokenService($issuer, $publicKey, $timeZone);
+
+        parent::setUp();
+
+    }
+    /**
+     * @test
+     * A valid token is provided and validated.  The test checks that the right sub value is returned.
+     */
+    public function it_accepts_a_good_token_and_returns_the_right_sub()

```

Comment:
Our naming convention for PHP functions is to use camelCase (for example, [testCreateUserDefaultRoles](https://github.com/GCTC-NTGC/gc-digital-talent/blob/main/api/tests/UserTest.php#L12)). I don't think this is documented anywhere, is it @tristan-orourke?

Reply:
Yeah, you're right.  All the rest of the code uses camelCase.  I've updated it.